core/service/src/client/mod.rs

```diff
@@ -6040,6 +6040,310 @@ pub(crate) mod tests {
         }
     }
 
+    #[cfg(feature = "insecure")]
+    #[tokio::test(flavor = "multi_thread")]
+    #[serial]
+    async fn default_insecure_dkg_backup() {
+        // NOTE: amount_parties must not be too high
+        // because every party will load all the keys and each ServerKey is 1.5 GB
+        // and each private key share is 1 GB. Using 7 parties fails on a 32 GB machine.
+
+        let amount_parties = 4;
+        let param = FheParameter::Default;
+        let dkg_param: WrappedDKGParams = param.into();
+
+        let key_id_1: RequestId = derive_request_id(&format!(
+            "default_insecure_dkg_backup_1_{amount_parties}_{param:?}",
+        ))
+        .unwrap();
+        let key_id_2: RequestId = derive_request_id(&format!(
+            "default_insecure_dkg_backup_2_{amount_parties}_{param:?}",
+        ))
+        .unwrap();
+
+        let test_path = None;
+        purge(test_path, test_path, test_path, &key_id_1, amount_parties).await;
+        purge(test_path, test_path, test_path, &key_id_2, amount_parties).await;
+        let (_kms_servers, kms_clients, internal_client) =
+            threshold_handles(*dkg_param, amount_parties, true, None, None).await;
+
+        // Purge the backup after booting the server to ensure nothing is backeup automatically during boot
+        purge_backup(test_path, amount_parties).await;
+        let keys_1 = run_threshold_keygen(
+            param,
+            &kms_clients,
+            &internal_client,
+            None,
+            &key_id_1,
+            None,
+            None,
+            true,
+        )
+        .await;
+        // check that we have the new mod switch key
+        let (client_key_1, _, server_key_1) = keys_1.clone().get_standard();
+        check_conformance(server_key_1, client_key_1);
+        let panic_res = std::panic::catch_unwind(|| keys_1.get_decompression_only());
+        assert!(panic_res.is_err());
+
+        let keys_2 = run_threshold_keygen(
+            param,
+            &kms_clients,
+            &internal_client,
+            None,
+            &key_id_2,
+            None,
+            None,
+            true,
+        )
+        .await;
+
+        // check that we have the new mod switch key
```

Comment:
```suggestion
```

don't think this keygen does this

Reply:
The test code was copied verbatim for `default_insecure_dkg` I did not check what actually happened in the conformance check. Regardless it is now removed.